README.md

```diff
@@ -48,23 +48,18 @@ for definition of maturity levels).
 Components marked `Stable` provide the following guarantees:
 
 - Field types, numbers and names will not change.
-- Numbers assigned to enum choices will not change.
 - Service names and service package names will not change.
-- Service operation names, parameter and return types will not change.
-
-The following changes are allowed:
-
-- Message names may change.
-- Enum names may change.
-- Enum choice names may change. This is allowed because enum choice names are not sent on
-  the wire.
-- The location of messages and enums, i.e. whether they are declared at the top
-  lexical scope or nested inside another message may change.
-- Package names may change.
-- Directory structure, location and the name of the files may change.
-
-Note that none of the above allowed changes affects the binary wire representation or the
-JSON wire representation.
+- Service method names will not change.
+- Service method parameter names will not change.
```

Comment:
Is generated code susceptible to breaking because of this?

Reply:
I think it would be a service request that failed in grpc due to a breaking naming change.

Reply:
Service names and method names are on the wire, so absolutely necessary for stability: https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md

I am not sure about parameter names. I will check.

Reply:
So, looks like parameter names are nowhere on the wire. However, I am not sure how to confirm they are not part of the generated API (and thus may be subject to symbolic stability). I could try going through all supported languages and see what the generated code looks like and whether parameter names matter, but that is going to be very difficult (and hard to confirm especially for dynamic languages where you can access parameter names at runime).

I suggest to play safe and just keep it here, unless someone has a better idea on how to confirm that we don't need this stability.